public/docs/ts/latest/quickstart.jade

```diff
@@ -10,479 +10,471 @@ include ../../../_includes/_util-fns
-    "suppressImplicitAnyIndexErrors":true
-    ```
+:marked
+  <a id="tsconfig"></a>
+  ### Appendix: TypeScript configuration
+  We added a TypeScript configuration file (`tsconfig.json`) to our project to
+  guide the compiler as it generates JavaScript files.
+  Get details about `tsconfig.json` from the official
+  [TypeScript wiki](https://github.com/Microsoft/TypeScript/wiki/tsconfig.json).
+
+  The options and flags in the file we provided are essential.
+  
+  We'd like a moment to discuss the `noImplicitAny` flag.
+  TypeScript developers disagree about whether it should be `true` or `false`.
+  There is no correct answer and we can change the flag later.
+  But our choice now can make a difference in larger projects so it merits
+  discussion.
+
+  When the `noImplicitAny` flag is `false`,
+  the compiler silently defaults the type of a variable to `any` if it cannot infer
+  the type based on how the variable is used. That's what we mean by "implicitly `any`".
+
+  When the `noImplicitAny` flag is `true` and the TypeScript compiler cannot infer
+  the type, it still generates the JavaScript files but
+  it also reports an error.
+
+  In this QuickStart and many of the other samples in this Developer Guide
+  we set the `noImplicitAny` flag to `false`.
+  
+  Developers who prefer stricter type checking should set the `noImplicitAny` flag to `true`.
+  We can still set a variable's type to `any` if
+  that seems like the best choice. We'd be doing so explicitly after
+  giving the matter some thought.
+
+  If we set the `noImplicitAny` flag to `true`, we may get implicit index errors as well.
+  If we feel these are more annoying than helpful,
+  we can suppress them with the following additional flag.
+  ```
+  "suppressImplicitAnyIndexErrors":true
+  ```
 
 .l-main-section
-  :marked
-    <a id="es6support"></a>
-    ### Appendix: Browser ES6 support
-    Angular 2 requires ES6 support, such as can be found in most modern
-    browsers. For older browsers (including IE 11) you can use a shim to get
-    the needed functionality.
-
-    After creating `package.json` (halfway through the quickguide), run this
-    command to add a shim to the project:
-
-  code-example(language="sh" format=".").
-    npm install es6-shim --save
-
-  :marked
-    Now you can load the shim in your `index.html` before the other scripts:
-
-  code-example(language="html" format=".").
-    &lt;script src=&quot;../node_modules/es6-shim/es6-shim.js&quot;&gt;&lt;/script&gt;
+:marked
+  <a id="systemjs"></a>
+  ### Appendix: SystemJS Configuration
+  
+  The QuickStart uses [SystemJS](https://github.com/systemjs/systemjs) to load application
+  and library modules. 
+  There are alternatives that work just fine including the well-regarded [webpack](https://webpack.github.io/).
+  SystemJS happens to be a good choice but we want to be clear that it was a choice and not a preference.
+  
+  All module loaders require configuration and all loader configuration 
```

Comment:
maybe link to SystemJS config docs https://github.com/systemjs/systemjs/blob/master/docs/config-api.md